src/Rector/Set/ValueObject/SetList.php

```diff
@@ -4,5 +4,5 @@
 
 final class SetList
 {
-    public const LAMINAS_SERVICEMANGER_40 = __DIR__ . '/../../../config/set/laminas-servicemanager-40.php';
-}
\ No newline at end of file
+    public const LAMINAS_SERVICEMANGER_40 = __DIR__ . '/../../../../config/rector/set/laminas-servicemanager-40.php';
```

Comment:
Btw, bugs like wrong paths like these can be discovered by PHPStan

See https://github.com/symplify/phpstan-rules/blob/main/docs/rules_overview.md#nomissingdirpathrule

Reply:
@TomasVotruba thank you 👍